.travis.yml

```diff
@@ -7,7 +7,16 @@ node_js:
   - "5"
   - "6"
 
-before_script: 'if [ -n "${BABEL-}" ]; then make bootstrap-babel ; fi'
+before_install:
+    # Rollup doesn't support node < 4.x. Switch to latest for build
+  - . $HOME/.nvm/nvm.sh
+  - nvm install stable && nvm use stable
```

Comment:
maybe should we use `nvm install --lts` instead of `nvm install stable` stable is deprecated first of all and is now an alias for node and second it currently installs node 7 and not sure how stable that is already. Or do you think thats okay?

And btw install also automatically calls use.